charts/gardener-extension-provider-gcp/templates/deployment.yaml

```
@@ -64,8 +68,14 @@ spec:
         - --webhook-config-namespace={{ .Release.Namespace }}
         - --webhook-config-service-port={{ .Values.webhookConfig.servicePort }}
         - --webhook-config-server-port={{ .Values.webhookConfig.serverPort }}
+        {{- if .Values.gardener.runtimeCluster.enabled }}
+        - --disable-controllers=backupentry,bastion,controlplane,infrastructure,worker,healthcheck
```

Improvement suggested by @timuthy 
```suggestion
        - --controllers=backupbucket,dnsrecord
```